### PR TITLE
Remove faker

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
Reimplement functionality we got from faker and avoid a dependency. Do we need a larger wordlist? Might just increase pw-length to be 4 or 5 words instead?

### CODE REVIEW

**Review by HerrEmil — APPROVED** [review]

File: src/browser/modules/Stream/Auth/ChangePasswordForm.jsx [hunk 1]
```diff
@@ -31,6 +31,11 @@ import Render from 'browser-components/Render'
 import InputEnterStepping from 'browser-components/InputEnterStepping/InputEnterStepping'
 import RevealablePasswordInput from './revealable-password-input'
 
+const getRandomWords = count =>
+  Array.from({ length: count }).map(
+    () => words[Math.floor(Math.random() * words.length)]
+  )
+
```
Comment (oskarhane) [inline src/browser/modules/Stream/Auth/ChangePasswordForm.jsx:38]: Maybe split this into a separate utils file

File: src/browser/modules/Stream/Auth/ChangePasswordForm.jsx [hunk 2]
```diff
@@ -31,6 +31,11 @@ import Render from 'browser-components/Render'
 import InputEnterStepping from 'browser-components/InputEnterStepping/InputEnterStepping'
 import RevealablePasswordInput from './revealable-password-input'
 
+const getRandomWords = count =>
+  Array.from({ length: count }).map(
+    () => words[Math.floor(Math.random() * words.length)]
```
Comment (oskarhane) [inline src/browser/modules/Stream/Auth/ChangePasswordForm.jsx:36]: Very nice solution!

File: package.json [hunk 3]
```diff
@@ -154,7 +154,6 @@
     "d3": "3",
     "dateformat": "^3.0.3",
     "deepmerge": "^2.1.1",
-    "faker": "^4.1.0",
```
Comment (oskarhane) [inline package.json:157]: 🙇

**Review by oskarhane — COMMENTED** [review]
I think it'd be good not to tie the random words function to the change password form file but to have it as a separate utils file (in the same folder).
And maybe write a test or two, just to verify and lock its functionality.

File: src/browser/modules/Stream/Auth/ChangePasswordForm.jsx (hunk 2 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Stream/Auth/ChangePasswordForm.jsx:36]: Thanks!

**Comment by OskarDamkjaer** [comment]
@oskarhane Thanks for the reminder to add tests, do have a quick look on the new changes! :)

File: src/browser/modules/Stream/Auth/utils.test.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+describe('getRandomWords', () => {
+  test('gives list of random words', () => {
+    const oneWord = getRandomWords(1)
+    const threeWords = getRandomWords(3)
+
+    expect(getRandomWords(0)).toEqual([])
+    expect(oneWord.length).toEqual(1)
+    expect(oneWord[0].length).toBeGreaterThan(2)
+    expect(threeWords.length).toEqual(3)
+    expect(threeWords.length).toEqual(3)
```
Comment (oskarhane) [inline src/browser/modules/Stream/Auth/utils.test.ts:12]: Duplication

**Review by oskarhane — APPROVED** [review]
Looks good to me except the duplicate assertion. 
Merge once that's gone and all tests are green!
Good stuff!

**Pull request merged**